pages/index.vue

```diff
@@ -791,15 +764,17 @@
             const date = new Date().toLocaleDateString();
             const time = new Date().toLocaleTimeString();
 
+            const url = new URL(this.url);
+
             // Addition of an entry to the history component.
             const entry = {
               label: this.requestName,
               status,
               date,
               time,
               method: this.method,
-              url: this.url,
-              path: this.path
+              url,
+              path: url.pathname
```

Comment:
On History section, `URL` contains `path`. Trim that down since `path` is already shown in separate field.

![Screen Shot 2019-10-15 at 10 49 24 AM](https://user-images.githubusercontent.com/10395817/66802446-10c50600-ef3a-11e9-9a74-7358ea0e5b6a.png)